_rules/audio-or-video-avoids-automatically-playing-audio-80f0bf.md

```diff
@@ -51,29 +53,32 @@ This rule applies to any `audio` or `video` element for which all the following
 - **autoplay**: the element has an `autoplay` [attribute value][] of `true`; and
 - **not muted**: the element has a `muted` [attribute value][] of `false`; and
 - **not paused**: the element has a `paused` [attribute value][] of `false`; and
-- **duration**: the element has a [media resource][] lasting more than 3 seconds and that contains audio.
+- **audio duration**: the element has a [media resource][] for which the audio output lasts more than 3 seconds.
```

Comment:
This seems slightly ambiguous to me. Should we do this based on the duration property?

Reply:
It does not work based on the duration because the media resource could have a duration longer than 3 seconds, while the audio in the media resource is shorter than 3 seconds. 

What exactly is ambiguous in the current description?